src/app/App.js

```diff
@@ -327,6 +328,11 @@ class App extends EventEmitter {
 			console.log('Link clicked outside app\'s base path');
 			return false;
 		}
+		if (this.isSamePendingNavigationPath_(path)) {
+			console.log('Discarding the navigation because is trying to navigate to the same path');
```

Comment:
I know there are already `console.log` all over the place in here, but it seems spammy... do we have a good justification for continuing this practice?

If you're going to keep it though, "because is trying" is not grammatical, you could abbreviate as "Discarding navigation to same path").

Reply:
We use [`stripDebug`](https://github.com/liferay/senna.js/blob/master/gulpfile.js#L12) to get rid of all of them in the production builds. Not that I'd advocate for this, but...

![just-sayin](https://user-images.githubusercontent.com/905006/90381152-e15cb980-e074-11ea-9d3d-d36d5ef95bbc.gif)

Reply:
Oh, a Gulp plug-in... how wonderful!

![dr-evil](https://user-images.githubusercontent.com/7074/90387135-eb3ae880-e085-11ea-83bb-6c09fd8fef31.gif)

Reply:
Towards stripDebug, We're using this babel plugin: https://github.com/liferay/senna.js/blob/2.x-stable/babelSennaPreset.js#L15 for removing console.log on prod builds 👀

Reply:
> We use stripDebug to get rid of all of them in the production builds.

![](https://media1.giphy.com/media/12NUbkX6p4xOO4/giphy.gif?cid=5a38a5a2946yk6cu7nfi2da4fbiy3hoim4adzw90k26jnlcs&rid=giphy.gif)